### PR TITLE
Improve verbosity levels

### PR DESCRIPTION
**Pull request by efritz**
Closes https://github.com/sourcegraph/lsif-go/issues/113.

### CODE REVIEW

**Review by gbrik — APPROVED** [review]
LGTM, especially the seconds elapsed after each step!

File: internal/indexer/progress.go [hunk 1]
```diff
@@ -10,97 +12,103 @@ import (
...
+}
+
+type Verbosity int
+
+const (
+	NoOutput Verbosity = iota
+	DefaultOutput
+	VerboseOutput
+	VeryVerboseOutput
+	VeryVeryVerboseOutput
```
[the remark this answers is not in the document]
Comment (efritz) [reply internal/indexer/progress.go:27]: When I have more to log 😛

File: cmd/lsif-go/index.go [hunk 2]
```diff
@@ -39,18 +39,38 @@ func writeIndex(repositoryRoot, projectRoot, moduleName, moduleVersion string, d
...
+			ShowAnimations: !noAnimation,
+		},
 	)
 
 	if err := indexer.Index(); err != nil {
 		return fmt.Errorf("index: %v", err)
 	}
 
-	if !noOutput && verboseOutput {
+	if !noOutput && verbosity > 0 {
```
Comment (efritz) [inline cmd/lsif-go/index.go:52]:
```suggestion
	if getVerbosity() >= VerboseOutput {
```

[the remark this answers is not in the document]
Comment (efritz) [reply cmd/lsif-go/index.go:52]: Good catch.

File: internal/indexer/progress.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Strum355) [reply internal/indexer/progress.go:27]: Start logging compliments to the user a la https://marketplace.visualstudio.com/items?itemName=saekiraku.rainbow-fart

**Pull request merged**